sahi/utils/file.py

```diff
@@ -169,7 +171,7 @@ def get_file_extension(path: str):
 def load_pickle(load_path):
     """
     Loads pickle formatted data (given as "data") from load_path
-    Example inputs:
+    Example inpts:
```

Comment:
@qingfengtommy this change can be reverted back :)

Reply:
Please check again. It should work...